### DISCUSSION TITLE
How to use EVE_PIN_PD_STATE?

### DISCUSSION THREAD

**Discussion by bla511** [RudolphRiedel/FT800-FT813#55] [Q&A] [answered]
Hello,

Just stumbled accross this define EVE_PIN_PD_STATE 0x71, has this been sent in the repository? I can't seem to find a way of using it. I see it i in the datasheet but just wondering if anyone knows how this works and how to implement?

Thanks,
Ben

**Comment by RudolphRiedel** [comment, marked as answer]
Oh, somehow I completely missed this.
EVE_PIN_PD_STATE  is for one of the host commands.
This are a different category alltogether and so the have their own function: EVE_cmdWrite(uint8_t command, uint8_t parameter)

EVE_cmdWrite(EVE_PIN_PD_STATE , 0x39);

I have never used this but looking at the datasheet I believe this is putting the Audio pin to Pull-Down during power down.

**Reply by bla511** [reply]
Thank you Rudolph, 

This makes sense, this may be useful for a bug I'm dealing with at the minute so having direct control over the pins on PD is quite handy.

Thanks again!
